spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -622,11 +617,17 @@ string or a `Claims` enum value. The string form would allow for specifying non-
 claims while the `Claims` enum approach guards against typos.
 
 #### Handling of Non-RequestScoped Injection of Claim Values
+
+MP-JWT implementations are required to support a claim value injection into an @ApplicationScoped scope. A warning may be issued when the @ApplicationScoped injection point is a primitive or wrapper type.
+
+MP-JWT implementations are required to generate a `javax.enterprise.inject.spi.DeploymentException` for a claim value injection into a @SessionScoped scope.
```

Comment:
We should use the term `Passivation capable beans, for example SessionScoped` to be entirely correct.

The sentence about `lifetime greater than @RequestScoped.` is not needed in that case.

Reply:
+1

Reply:
@rdebusscher Sure, so you mean to have something like
```
MP JWT implementations may issue a warning for Passivation capable beans, for example, SessionScoped context.
```
Then I'll drop a requirement to throw an exception if we allow for a warning in case of SessionScoped ? It would be my preference for sure

Reply:
not the warning, the exception when used on a Passivation capable bean.

Reply:
@rdebusscher Please check the updated text...